doc/src/stdlib/arrays.md

```diff
@@ -164,16 +164,16 @@ Base.mapslices
 ## Combinatorics
 
 ```@docs
-Base.Random.randperm
-Base.Random.randperm!
+Random.randperm
```

Comment:
Shouldn't these move to the random docs instead?

Reply:
I don't know. As long as `Random` is bundled with `Base`, it may be useful to have that here as before?

Reply:
I ended up following your suggestion :)